tardis/tardis_portal/static/css/default.css

```diff
@@ -95,6 +95,11 @@ div.copylink {
 .datafile_name {
     font-size: 110%;
     font-weight: bold;
+    word-break: break-all;
+}
+
+.datafiles.table {
+    width: auto !important;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/mytardis/mytardis/file/1298047134/issues/source?bid=2125231&fileBranchId=2948397#l102)